lua/nvim-treesitter/statusline.lua

```diff
@@ -13,9 +13,9 @@ function M.statusline(opts)
     return
   end
   local options = opts or {}
-  -- if type(opts) == "number" then
-  --   options = { indicator_size = opts }
-  -- end
```

Comment:
Why was this already there, but commented out? Are there any unwanted consequences that this may re-add that were the reason for it being commented out originally?

Reply:
As documented [here](https://github.com/nvim-treesitter/nvim-treesitter#statusline-indicator), the `statusline` can accept number as opts, and the commit lead to comment it: [fix(statusline): adjust to the new API (query)](https://github.com/nvim-treesitter/nvim-treesitter/commit/2490a8bbcaca5c84b86ea82661854e1cfb865571#diff-73d8fd4bc707f46576768511e62a4a2ddc0bb32f1e676b4b208e9d61e193d8acL51)

Reply:
I'm not aware of any risk of enabling this again. Should we just try it?